onnxscript/test/models/graph_attr.py

```diff
@@ -0,0 +1,34 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from onnxscript import graph, script
+from onnxscript.onnx_opset import opset15 as op
+from onnxscript.onnx_types import BOOL, INT64
+
+
+@script()
+def cumulative_sum(X: INT64['N']):
+    '''Test use of a nested-function as a graph-attribute, using the Scan operator.'''
```

Comment:
```suggestion
    """Test use of a nested-function as a graph-attribute, using the Scan operator."""
```

Reply:
Even though test models are not formatted, if code format is not part of what is tested, I would run black on these files.